app/merginapi.cpp

```diff
@@ -11,14 +11,18 @@
 
 MerginApi::MerginApi(const QString &root, const QString& dataDir, QObject *parent)
   : QObject (parent)
-  , mApiRoot(root)
   , mDataDir(dataDir + "/")
   , mCacheFile(".projectsCache.txt")
 {
     QObject::connect(this, &MerginApi::syncProjectFinished,this, &MerginApi::setUpdateToProject);
     QObject::connect(this, &MerginApi::merginProjectsChanged,this, &MerginApi::cacheProjects);
     QObject::connect(this, &MerginApi::authChanged,this, &MerginApi::saveAuthData);
 
+    QSettings settings;
+    settings.beginGroup("Input/");
+    settings.setValue("apiRootDefault", root);
+    settings.endGroup();
+
```

Comment:
let's just initialize mApiRoot to MerginApi::defaultApiRoot() here to make it simpler

Reply:
Since defualtApiRoot is hardcoded, this part is obsolete. Anyway, wouldn't be better to load apiRoot from QSettings and only load default if its empty?

Reply:
Ah right - let's just drop this.

I see in loadAuthData() we already load apiRooot - that's good. Yes let's load api root from qsettings and only use default when it was not set yet